composer.json

```diff
@@ -12,38 +12,42 @@
         {
             "name": "Samuel Georges",
             "email": "daftspunky@gmail.com"
+        },
+        {
+            "name": "Luke Towers",
+            "email": "octobercms@luketowers.ca"
         }
     ],
     "require": {
-        "php": ">=7.0",
-        "doctrine/dbal": "2.5.*",
-        "erusev/parsedown-extra": "~0.7",
-        "kriswallsmith/assetic": "~1.3",
+        "php": "^7.2",
+        "doctrine/dbal": "^2.6",
+        "erusev/parsedown-extra": "~0.8",
+        "assetic/framework": "dev-2.0/dev",
         "linkorb/jsmin-php": "~1.0",
-        "leafo/scssphp": "~0.6",
-        "symfony/yaml": "~3.2",
+        "symfony/yaml": "^3.4",
         "twig/twig": "~2.0",
         "league/csv": "~8.0",
         "jenssegers/date": "~3.0",
-        "laravel/tinker": "~1.0"
+        "laravel/tinker": "^1.0",
+        "laravel/helpers": "^1.0",
+        "fideloper/proxy": "^4.0"
```

Comment:
What is this dependency used for?

Reply:
@LukeTowers it's a trusted proxy library. It's required for Laravel 5.6, but it's only required by `laravel/laravel`, even though the framework uses it. So I had to make it a dependency for our library too.

Reply:
@bennothommo could you point me to where it's required by the framework?

Reply:
@LukeTowers I was mistaken - for some reason, in my initial tests last year, it was erroring out if `fideloper/proxy` was missing. When I've done a fresh install, everything works now. The framework doesn't use it - only the `laravel/laravel` package does.